package.json

```diff
@@ -16,6 +16,7 @@
     "core-js": "^3.6.4",
     "jsonwebtoken": "^8.5.1",
     "konva": "^4.0.18",
+    "lodash": "^4.17.15",
```

Comment:
@bryansamuel1 required?
@Eirmas Comment?

Reply:
not the whole library no. Please only add the packages that contain specific lodash functions. This for example is the one for https://www.npmjs.com/package/lodash.range lodash range function